### PR TITLE
J0190UIL-1144 Update link to partial buffer demo

### PR DESCRIPTION
**Pull request by apoudret-microej**

### CODE REVIEW

File: ApplicationDeveloperGuide/UI/MWT/concepts.rst [hunk 1]
```diff
@@ -50,7 +50,7 @@ A render policy is a strategy that MWT uses in order to repaint the entire deskt
...
 
-When using a :ref:`partial buffer <section_display_partial_buffer>`, these render policies can not be used because they render the entire screen in a single pass. Instead, a custom render policy which renders the screen in multiple passes has to be used. Refer to the `partial buffer demo <https://github.com/MicroEJ/>`__ for more information on how to implement this render policy and how to use it.
+When using a :ref:`partial buffer <section_display_partial_buffer>`, these render policies can not be used because they render the entire screen in a single pass. Instead, a custom render policy which renders the screen in multiple passes has to be used. Refer to the `partial buffer demo <https://github.com/MicroEJ/Partial-Buffer-Demo>`__ for more information on how to implement this render policy and how to use it.
```
Comment (frivieremicroej) [inline ApplicationDeveloperGuide/UI/MWT/concepts.rst:53]: https://github.com/MicroEJ/Partial-Buffer-Demo
this repo does not yet exist, but warning: why it shall not respect the convention https://github.com/MicroEJ/Demo-XXX ?

Comment (apoudret-microej) [reply ApplicationDeveloperGuide/UI/MWT/concepts.rst:53]: do we have this convention? this is not correct english.. i think we should say "widget demo" and not "demo widget" for example

Comment (frivieremicroej) [reply ApplicationDeveloperGuide/UI/MWT/concepts.rst:53]: @gharelMicroEJ please point to the convention

Comment (apoudret-microej) [reply ApplicationDeveloperGuide/UI/MWT/concepts.rst:53]: @frivieremicroej he is on holidays

Comment (frivieremicroej) [reply ApplicationDeveloperGuide/UI/MWT/concepts.rst]: @apoudret-microej please update with https://github.com/MicroEJ/Demo-PartialBuffer

**Pull request merged**